pages/common/swagger-codegen.md

```diff
@@ -0,0 +1,19 @@
+# swagger-codegen
+
+> Generate code and documentation for your REST api from a OpenAPI/swagger definition
+
+- Generate from a file or an url
+
+`swagger-codegen generate --input-spec {{url_or_file}} ---lang {{language}} --library {{library}}`
```

Comment:
I don't see `--library` being a required option for this command. If it is not common enough, we should remove it. Else, the first example should just have input_spec and lang, and then another one with library.

Reply:
It's an important option. I modified to have two examples